public/scripts/routes.js

```diff
@@ -0,0 +1,5 @@
+page('/', indexController.index);
+page('/index', indexController.index);
```

Comment:
These first two lines could alternatively be written as, first a redirect to '/index', and then a callback on the index route.
``` javascript
page('/', '/index');
page('/index', indexController.index);
```